packages/shared/ReactFeatureFlags.js

```diff
@@ -123,3 +123,6 @@ export const enableLegacyFBSupport = false;
 // interleaved event. Remove this flag once we have migrated to the
 // new behavior.
 export const deferRenderPhaseUpdateToNextBatch = true;
+
+// Replacement for runWithPriority in React internals.
+export const enableCurrentLanePriority = false;
```

Comment:
You can put this ReactFeatureFlags.www-dynamic so that tests run against both values:

https://github.com/facebook/react/blob/8f05f2bd6d131a39835d468622e248b231ccbf8e/packages/shared/forks/ReactFeatureFlags.www-dynamic.js#L10-L20